tests/tests/test_distance/test_distance.py

```diff
@@ -0,0 +1,39 @@
+"""
+"""
+import phoebe
+
+
+def test_distance_scaling_alt_backend(verbose=False, plot=False):
+    b = phoebe.Bundle.default_binary()
+
+    b.add_dataset('lc', compute_phases=[0.25])
+    for backend in phoebe.list_available_computes():
+        if backend == 'phoebe':
+            # compute options already exist
+            continue
+        if backend == 'jktebop':
+            # TODO: re-enable this in the test, currently fails on CI
```

Comment:
does this limit the use of jktebop for distances?

Reply:
no, it just means we don't have test coverage.  The problem is with _running_ jktebop on CI, not necessarily the distance treatment.